src/_base/harness/scripts/xdebug_version_sync.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+PREPARE_CONFIG=0
+RESTART_FPM=0
+
+xdebug_version_sync()
+{
+  INSTALLED_VERSION=$(docker-compose exec -T -u root "$1" pecl info xdebug 2>&1 | grep "Release Version" | awk '{print $3}')
+  INSTALLED_MAJOR_VERSION=$(echo "$INSTALLED_VERSION" | cut -d '.' -f1)
```

Comment:
```suggestion
  local INSTALLED_MAJOR_VERSION
  INSTALLED_MAJOR_VERSION="$(echo "$INSTALLED_VERSION" | cut -d '.' -f1)"
```

Reply:
Have added these, and also used `local` on the `INSTALL_CANDIDATE` variable - is that OK too?

Reply:
Perfect :)
Stops the global scope being affected